runtime/bcutil/ROMClassBuilder.cpp

```diff
@@ -429,12 +429,12 @@ ROMClassBuilder::handleAnonClassName(J9CfrClassFile *classfile, ROMClassCreation
 	/* Mark if the class is a Lambda class. */
 #if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
 	if (!context->isLambdaFormClass()
-		&& isLambdaClassName(reinterpret_cast<const char *>(_anonClassNameBuffer), newAnonClassNameLength - 1)
+		&& isLambdaClassName(reinterpret_cast<const char *>(_anonClassNameBuffer), newAnonClassNameLength - 1, NULL)
```

Comment:
Maybe toss in an inline `/* prefixLength */` comment before the `NULL` just for clarity. Ditto below.